.github/workflows/ci.yml

```diff
@@ -40,10 +39,8 @@ jobs:
         cabal sdist -z -o .
         cabal get bytestring-*.tar.gz
         cd bytestring-*/
-        cabal build -w /opt/ghc/bin/ghc-${{ matrix.ghc }} all
-    - name: Test
-      working-directory: ./tests
-      run: cabal test -w /opt/ghc/bin/ghc-${{ matrix.ghc }} --test-show-details=direct
+        cp ../cabal.project .
+        cabal test -w /opt/ghc/bin/ghc-${{ matrix.ghc }} --test-show-details=direct
```

Comment:
This mix of of testing the sdist contents with the original `cabal.project` config is a bit weird IMHO but probably fine.

How about renaming the job from `Build` to `Test` though, since running the tests is the main thing it does?!

Reply:
Done.